cpp/src/arrow/util/bpacking_avx512_codegen.py

```diff
@@ -0,0 +1,153 @@
+#!/bin/python
+
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+# Usage: python bpacking_avx512_codegen.py > bpacking_avx512_generated.h
+
+def print_unpack_bit_func(bit):
+	shift = 0
```

Comment:
Can you make sure this file is formatted according to [PEP 8](https://www.python.org/dev/peps/pep-0008/).
Mostly, use 4 spaces for indentation instead of tabs.
You may automate this using e.g. [autopep8](https://pypi.org/project/autopep8/).

Reply:
thx, fixed.